### ISSUE TITLE
Workflow overview area not showing records that are submitted for review

### ISSUE THREAD

**Issue by charliestrafe** [engram-design/Workflow#152]
**Description**
The workflow overview area (admin/workflow), where reviewers and publishers can review what has been submitted for review is not showing any results.
Whats weird is that it is showing the correct count at the bottom of the table, however the table itself is blank.

<img width="1427" alt="Screenshot 2022-07-04 at 09 15 46" src="https://user-images.githubusercontent.com/47243887/177112821-b48e54b9-b7a3-4e50-911f-32f2324b6b4e.png">

I have tried this as both a reviewer and publisher user.
I have also tried this as a craft Admin user.

**Steps to reproduce**
1. Submit an item for review as an editor
2. View the workflow overview area, the table will be blank.

This has only been since applying one of the recent versions.

**Additional info**
- Plugin version:
- Craft version: dev-craft-4 as 2.0.0-beta.4 and 2.0.0-beta.4 (tried both and still doesn't work)
- Multi-site: No

**Comment by charliestrafe** [comment]
Hi, I'm still struggling with this, could someone help me out? Thanks!

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by charliestrafe** [comment]
Thank you this has fixed this issue

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.5](https://github.com/verbb/workflow/releases/tag/2.0.0-beta.5)
